src/devices/bus/msx/cart/video80.cpp

```diff
@@ -0,0 +1,201 @@
+// license:BSD-3-Clause
+// copyright-holders:F. Ulivi
+
+/*********************************************************************
+
+    VIDEO80 homebrew 80-column video card
+
+    This is a 80-column video card that I designed and built in 1988.
+    Here's a summary of its features:
+    - 80 columns by 24 rows of text video
+    - 8x8 character cell
+    - 16 MHz dot clock, 50 Hz frame rate
+    - Based on MC6845 CRT controller
+    - 4k of SRAM: half for the framebuffer & half for the character
+      generator.
+    - 4k of EPROM: it contains the firmware and the compressed font.
+      FW supports seamless operation in both BASIC and MSXDOS
+      environments. No driver or additional sw is needed.
+
+*********************************************************************/
+
+#include "emu.h"
+#include "video80.h"
+#include "cpu/z80/z80.h"
+#include "video/mc6845.h"
+
+#include "emupal.h"
+#include "screen.h"
+
+// Debugging
+#undef VERBOSE
+#define VERBOSE 0
+#include "logmacro.h"
```

Comment:
Why you need to `#undef VERBOSE` like at all?

Reply:
I don't know, it's been carried over by copy-paste since forever...